src/main/resources/application.yml

```diff
@@ -41,12 +41,6 @@ spring:
 
   flyway:
     locations: classpath:/db/migration/postgres,classpath:/db/migration
-    url: ${APP_DB_URL}
-    user: ${DB_USERNAME}
-    password: ${DB_PASSWORD}
-    placeholders:
-      database_password: ${DB_PASSWORD}
-    schemas: ${DB_NAME}
```

Comment:
Does it not need any?  will it use the existing database config?

Reply:
Running locally it did.  I am still not 100% it is quite right though.  So Spring (automagically) integrates with flyway i.e. it should pick these up as needed (in theory).